invenio_curations/tasks.py

```diff
@@ -7,21 +7,24 @@
 # under the terms of the MIT License; see LICENSE file for more details.
 
 """Celery tasks for curations."""
+from typing import Any
 
 from celery import shared_task
 from flask import current_app
 from invenio_access.permissions import system_identity
 
+from invenio_curations.services import CurationRequestService
 from invenio_curations.services.errors import OpenRecordCurationRequestAlreadyExists
 
-from .proxies import current_curations_service
+from .proxies import current_curations_service, unproxy
 
 
 @shared_task(ignore_result=True)
-def request_record_curation(topic, user_id):
+def request_record_curation(topic: Any, user_id: str) -> None:
```

Comment:
Do you know what type this should be? It seems like it should be RDMDraft or RDMRecord?